backend/src/main/java/com/yigongil/backend/config/SwaggerConfig.java

```diff
@@ -0,0 +1,32 @@
+package com.yigongil.backend.config;
+
+import io.swagger.v3.oas.models.Components;
+import io.swagger.v3.oas.models.OpenAPI;
+import io.swagger.v3.oas.models.info.Info;
+import io.swagger.v3.oas.models.security.SecurityScheme;
+import io.swagger.v3.oas.models.security.SecurityScheme.Type;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class SwaggerConfig {
+
+    @Bean
+    public OpenAPI publicApi() {
+        Info info = new Info()
+                .title("201 Created")
+                .description("201 Created의 Open Api입니다.")
+                .version("v1");
+
+        SecurityScheme securityScheme = new SecurityScheme()
+                .type(Type.HTTP)
+                .scheme("Bearer")
+                .name("github");
```

Comment:
p4: 클라이언트 입장에선 github 토큰을 보내기보다는 JWT 토큰을 보내는 것이니 다음과 같이 수정하면 어떨까요??
[레퍼런스1](https://karismamun.tistory.com/80), [레퍼런스2](https://www.baeldung.com/openapi-jwt-authentication) 남깁니다.

```suggestion
        SecurityScheme securityScheme = new SecurityScheme()
                        .name("Authorization")
                        .type(Type.HTTP)
                        .in(HEADER)
                        .bearerFormat("JWT")
                        .scheme("Bearer");
```

* 근데 바꾸고 실행해보니 클라이언트 입장에서는 달라지는 게 없는 거 같네요... 저는 Authorization 헤더에 넣어야 된다는 힌트를 주고 싶었는데 이 코드가 그 역할을 하진 못하네요 ㅎㅎ;;
수정할지 말지 여부는 주드가 선택해주세용~

Reply:
괜찮은 방식같네요 수정할게요